pkg/controller/clusterinst_api.go

```diff
@@ -1043,10 +939,19 @@ func (s *ClusterInstApi) createClusterInstInternal(cctx *CallContext, in *edgepr
 			}
 		}
 
-		in.IpAccess, err = validateAndDefaultIPAccess(ctx, in, cloudlet.PlatformType, features, cb)
+		err = validateAndDefaultIPAccess(ctx, in, cloudlet.PlatformType, features)
 		if err != nil {
 			return err
 		}
+		// TODO: Networks need to be made independent of cloudlets, as
+		// developers do not have direct visibility into cloudlets.
+		// Rather than developers referencing specific networks created by
+		// the operator, developers should specify the requirements for a
+		// pre-existing operator-created network (such as private, public,
+		// etc) that can then be matched against pre-existing networks.
+		// Or, more likely, the requirements for a developer-specific
+		// network that can be created dynamically and is dedicated to the
+		// developer.
```

Comment:
Not sure I follow this comment. Do you mean that networks should not be exposed to the developers? Right now a `network` object cannot be created by a developer, but I don't think we have a good grasp of what it should be to begin with right now.

Reply:
Let me fix up the comment. The main problem right now is that networkKey depends on cloudletKey, but developers can't see cloudlets anymore - they're hidden by zones. So there's no way a developer could (or should) be able to specify a network.